### PR TITLE
Business info wizard form

### PR DESCRIPTION
**Pull request by jakemer**
This branch adds the `BusinessFormStepped` component, which provides a "wizard" style layout, previous/next buttons with stepping functionality, and a progress indicator.

It also adds some consistency to business form subcomponents in regards to their spacing (in the normal form at least) and their header styling.

**It does not submit data yet,** that will come in a subsequent PR for the purpose of keeping code reviews smaller and more focused.

Steps for Testing/QA
--------------------

### CODE REVIEW

**Review by colganmeanor — DISMISSED** [review]
Looks like everything is shaping up nicely here - great work so far!

File: stencil-library/src/components/business-form/business-form-stepped.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  tag: 'justifi-business-form-stepped',
+  styleUrl: 'business-form-stepped.scss',
+})
+export class BusinessFormStepped {
+  @Prop() authToken: string;
+  @Prop() businessId?: string;
+  @State() isLoading: boolean = false;
+  @State() currentStep: number = 0;
+  @State() totalSteps: number = 4;
+  @State() percentageComplete: number = 0;
```
Comment (TiagoFuelber) [inline stencil-library/src/components/business-form/business-form-stepped.tsx:32]: I suggest refactoring percentageComplete from a state to a computed getter for simplicity and maintainability:

```
get percentageComplete() {
  return (this.currentStep / this.totalSteps) * 100;
}

```

This change will let us:

Remove the @Watch on currentStep.
Drop the updatePercentageCompleteOnStepChange method.
Also drop the percentageComplete state.
Let's keep derived values like this computed to ensure accuracy.

Thoughts?

**Review by TiagoFuelber — APPROVED** [review]
Looks very good, just one comment!

File: stencil-library/src/components/business-form/business-form-stepped.tsx (hunk 1 above)
Comment (jakemer) [reply stencil-library/src/components/business-form/business-form-stepped.tsx:32]: Sure, good suggestion!

**Review by TiagoFuelber — COMMENTED** [review]
Nice! Looks good

**Pull request merged**